src/components/Form/index.js

```diff
@@ -81,6 +81,7 @@ export const Form = ({
         update: true,
         send_application: true, // Enable edition for new enrollment (ie. enrollment has no id)
       },
+      demarche: 'default',
```

Comment:
always in default state ? Or only when demarches is defined ?

Reply:
committed by error! nicely spotted Xav Eagle-eyed Jp!